mifospay/src/main/java/org/mifos/mobilewallet/mifospay/auth/ui/LoginActivity.java

```diff
@@ -214,4 +222,13 @@ public void onComplete(@NonNull Task<Void> task) {
                     });
         }
     }
+
+    private void hideSoftKeyboard() {
```

Comment:
@anubhav11march Thanks for the PR! 
Can you use the `hideSoftKeyboard()` method available in the utils instead of creating it again